src/ts/server/s3.ts

```diff
@@ -1,13 +1,30 @@
-import { R2_ACCESS_KEY, R2_ACCOUNT_ID, R2_SECRET_KEY } from '$env/static/private';
+import { R2_ACCESS_KEY, R2_ACCOUNT_ID, R2_PUBLIC_URL, R2_SECRET_KEY } from '$env/static/private';
 import { S3Client } from '@aws-sdk/client-s3';
+import { dev } from '$app/environment';
 
-const s3 = new S3Client({
-	endpoint: `https://${R2_ACCOUNT_ID}.r2.cloudflarestorage.com`,
-	region: 'auto',
-	credentials: {
-		accessKeyId: `${R2_ACCESS_KEY}`,
-		secretAccessKey: `${R2_SECRET_KEY}`
+function getS3Client(): S3Client {
+	if (dev) {
```

Comment:
The S3 Client using the Localstack should be run only during the dev. If there is other way to configure that here let me know.